RELEASE.md

```diff
@@ -0,0 +1,16 @@
+# Release
```

Comment:
I think it would be valuable to include the full release process here, i.e. what needs to be done with the cosmos-sdk fork for example.

Reply:
added.